src/gallery/_gallery.scss

```diff
@@ -0,0 +1,97 @@
+.gallery {
+  position: relative;
+  overflow: hidden;
+
+  &:hover {
+    .gallery-controls {
+      opacity: 1;
+    }
+  }
+}
+
+.gallery-slider {
+  width: 100%;
+  height: 100%;
+  white-space: nowrap;
+}
+
+// @@@ [1] inflexible positioning and size should not be determined
+// by the component. let each instance define their own or set
+// overideable defaults
+.gallery-controls {
+  @include z-index(gallery-controls);
+
+  @include square(65px);
+
+  border-radius: 33px;
+  transition: opacity 0.2s ease, background-color 0.15s;
+  position: absolute;
+  top: 50%;
+  margin-top: -32.5px;
+  right: 25px;
+  text-align: center;
+  cursor: pointer;
+  opacity: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+
+  .touch & {
+    display: none;
+  }
+
+  [data-icon="arrow"] {
+    vertical-align: top;
+    margin-top: 24px;
+    fill: color(background);
+    transform: rotate(0deg);
+  }
+
+  #{$focus} {
+    background-color: rgba(0, 0, 0, 0.2);
+  }
+}
+
+.gallery-prev {
+  left: 25px;
+
+  [data-icon="arrow"] {
+    margin-right: 2px;
+    transform: rotate(180deg);
+  }
+}
+
+.gallery-slide {
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+}
+
+.gallery-slide-image {
+  width: 100%;
+  cursor: pointer;
+}
+
+.gallery-dots {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  text-align: center;
+  cursor: pointer;
+}
+
+.gallery-dot {
+  @include circle(8px, primary);
+
+  @include active-opacity(0.5);
+
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0.3);
+  transition: opacity 0.2s ease;
+  margin: 6px;
+  display: inline-block;
+  background-color: rgba(0, 0, 0, 0.4);
+  background-size: contain;
+}
+
+.gallery-slide-wrapper {
+  @include square(100%);
+}
```

Comment:
@misscs I think it's being used on the storefront static pages, but I'm gonna strip all this out later when I use the new gallery module on those (once it's working with images).